### ISSUE TITLE
Channel map causes exception on iOS

### ISSUE THREAD

**Issue by andriyvnester** [widok/widok#108]
While simple code like this works fine:
```scala
object AnApp extends PageApplication {
	val var1 = Var[String]("value")

	def view() = span(var1)
	def ready() = ()
}
```
this code throws exception on iOS (iOS 8 tested, any browser):
```scala
object AnApp2 extends PageApplication {
	val var1 = Var[String]("value")

	def view() = var1.map(span(_))
	def ready() = ()
}
```
```
InvalidCharacterError: DOM Exception 5: An invalid or illegal character was specified, such as in an XML name.
```

**Comment by tindzk** [comment]
Thanks for reporting! I have committed a tentative fix. Does it resolve your problem?

**Comment by andriyvnester** [comment]
Yes, it does. Thank you for the fix.

It was also an issue on default browser for Android older than 4.4. It is resolved now as well (Android 4.1.1 tested).
